stacks_queues/queue.py

```diff
@@ -1,3 +1,4 @@
+from linked_list import LinkedList;
```

Comment:
Need to specify directory, or python can't find the file

```suggestion
from stacks_queues.linked_list import LinkedList;
```